config.php.example

```diff
@@ -1,6 +1,11 @@
 <?php
-
-
+$config_file = __DIR__.'/config/config.json';
+if (!file_exists($config_file)) {
+    header("Location: /capture/config_tcat.php");
```

Comment:
What happens in the CLI scripts, e.g. `controller.php`? It might e.g. make sense to exist with a non-zero exit code in that case, instead of redirecting.

Reply:
Now `exit(1)` is used.